src/main/java/io/strimzi/kafka/bridge/http/HttpSinkBridgeEndpoint.java

```diff
@@ -615,6 +614,18 @@ private MessageConverter<K, V, byte[], byte[]> buildMessageConverter() {
         return null;
     }
 
+    private String getContentType() {
+        switch (this.format) {
+            case JSON:
+                return BridgeContentType.KAFKA_JSON_JSON;
+            case BINARY:
+                return BridgeContentType.KAFKA_JSON_BINARY;
+            case TEXT:
+                return BridgeContentType.KAFKA_JSON_TEXT;
+        }
+        return null;
```

Comment:
This should be something like ... `throw new IllegalArgumentException();`. It should never happen but returning `null` doesn't even make any sense.